pages/common/content/index.js

```diff
@@ -9,7 +9,8 @@ module.exports = {
       dashboard: '{{establishment.name}}',
       read: 'Details',
       update: 'Amend establishment details',
-      apply: 'Apply for licence'
+      apply: 'Apply for licence',
+      fees: 'Projected licence fees'
```

Comment:
```suggestion
      fees: 'Estimated licence fees'
```
In the S&T they asked not to use "Projected" to avoid confusion with Projects.

Reply:
"Forecasted" might work also to avoid confusion between Estimated / Establishment 😄

Reply:
Will change these in the future once someone has taken a position on what it _should_ be. It's going to be a long way off from being visible to external users anyway.